server/flaskr/services/inputs/input_validator.py

```
@@ -44,9 +59,24 @@ def validate(self, value) -> InputError:
         return (
             InputError.NoError
             if re.match(self.EmailRegex, value) is not None
-            else InputError.EmailFormatInvalid
+            else InputError.InvalidFormat
         )
 
+class PasswordValidator(StringValidator):
+    PasswordRegex = r"^(?=.*[A-Za-z])(?=.*\d)(?=.*[!@#$%^&*()\-_=+])[A-Za-z\d!@#$%^&*()\-_=+]+$"
+
+    def __init__(self):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)